modules/tinymce/src/core/main/ts/fmt/DefaultFormats.ts

```diff
@@ -154,6 +154,7 @@ const get = function (dom: DOMUtils) {
     hilitecolor: { inline: 'span', styles: { backgroundColor: '%value' }, links: true, remove_similar: true, clear_child_styles: true },
     fontname: { inline: 'span', toggle: false, styles: { fontFamily: '%value' }, clear_child_styles: true },
     fontsize: { inline: 'span', toggle: false, styles: { fontSize: '%value' }, clear_child_styles: true },
+    lineheight: { selector: 'h1,h2,h3,h4,h5,h6,p', defaultBlock: 'p', styles: { lineHeight: '%value' }},
```

Comment:
Should we include `div` in this list?

Reply:
What about `li` as they also can contain paragraphs of text without technically be a paragraph? Also maaaaybe `td`/`th` because of the same reason?

Reply:
Adding all of these. If there's anything else that needs to go into this list speak now or ~forever hold your peace~ do it in another PR